x.toml

```diff
@@ -262,6 +262,11 @@ mark-changed = "all"
 globs = ["developer-docs-site/**/*", "documentation/**/*", "docker/**/*", "language/documentation/**/*", "specifications/**/*", "scripts/**/*", "terraform/**/*"]
 mark-changed = []
 
+[[determinator.path-rule]]
+# Ignore typescript sdk
+globs = [".ecosystem/typescript/**/*"]
```

Comment:
Still trying to figure out how this `x.toml` works 😄 But, why `.ecosystem` and not `ecosystem`?

Reply:
a typo :-(